CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Publish empty metadata event and empty contact list on delete account. [#1530](https://github.com/planetary-social/nos/issues/1530)
 
 ### Internal Changes
+- Update Xcode to version 16.0.
```

Comment:
I'd love a little more explanation in here since we're changing `.xcode-version` to 15.4 while adding compatibility for 16. Something along those lines would be perfect.

Reply:
@mplorentz I did it. I also moved it up to the Unreleased section.